src/main/resources/application.yml

```diff
@@ -11,6 +11,10 @@ spring:
       uris: http://elasticsearch:9200
   kafka:
     bootstrap-servers: kafka:9092
+  datasource:
+    url: jdbc:postgresql://localhost:5432/okapi_modules
```

Comment:
Have you had a chance to verify it on a remote environment? It looks like that it won't work,